Source/Language/ISetupSequentialResult.cs

```diff
@@ -18,6 +18,18 @@ public interface ISetupSequentialResult<TResult>
 		/// </summary>
 		ISetupSequentialResult<TResult> Returns(TResult value);
 
+		/// <summary>
+		/// Uses delegate to get return value
+		/// </summary>
+		/// <returns></returns>
+		ISetupSequentialResult<TResult> Returns(Delegate valueFunction);
```

Comment:
As above, I'd say this overload isn't necessary at all, it doesn't make sense to pass any `Delegate`. Moq won't know how to invoke *any* arbitrary delegate.